homeassistant/components/light/hue.py

```diff
@@ -47,7 +47,7 @@
 
 PHUE_CONFIG_FILE = 'phue.conf'
 
-SUPPORT_HUE_ON_OFF = (SUPPORT_FLASH | SUPPORT_TRANSITION | SUPPORT_FLASH)
+SUPPORT_HUE_ON_OFF = (SUPPORT_FLASH | SUPPORT_TRANSITION)
```

Comment:
I'm fixing only the nonsensical "SUPPORT_FLASH | SUPPORT_FLASH" for SUPPORT_HUE_ON_OFF